pathspider/cmd/measure.py

```diff
@@ -107,6 +108,10 @@ def run_measurement(args):
                 outputfile.write(json.dumps(result) + "\n")
                 logger.debug("wrote a result")
                 spider.outqueue.task_done()
+        #upload data if option is set
```

Comment:
This is a redundant comment. Anything that is obvious from reading the code shouldn't have comments, it just makes for clutter.